modin/core/io/text/csv_glob_dispatcher.py

```diff
@@ -64,10 +64,18 @@ def _read(cls, filepath_or_buffer, **kwargs):
                     + f"files at once. Did you forget it? Passed filename: '{filepath_or_buffer}'"
                 )
             if not cls.file_exists(filepath_or_buffer, kwargs.get("storage_options")):
-                return cls.single_worker_read(filepath_or_buffer, **kwargs)
+                return cls.single_worker_read(
+                    filepath_or_buffer,
+                    cls._file_not_found_msg(filepath_or_buffer),
```

Comment:
`filepath_or_buffer` can be types other than strings so we should be careful of that in `_file_not_found_msg()`.

Reply:
Any suggestions on how best to handle the non-string case? Would printing the `repr` or `str` be acceptable?

Reply:
I think working with the `str` representation should be good enough since that is usually supported by most path objects. However, I would check with other folks and see what they have to say.

cc: @modin-project/modin-contributors

Reply:
It looks like in this particular branch, the filename will always be a string. In other branches, I misunderstood the behavior -- if `not cls.pathlib_or_pypath(filepath_or_buffer)` holds, then that means the argument should be a buffer; I've now updated the error message in this branch accordingly.

In any case, the f-string should automatically interpolate the `str` representation of the argument.